src/wp-includes/widgets/class-wp-nav-menu-widget.php

```diff
@@ -114,15 +114,27 @@ public function update( $new_instance, $old_instance ) {
 	 */
 	public function form( $instance ) {
 		global $wp_customize;
-		$title = isset( $instance['title'] ) ? $instance['title'] : '';
+		$title    = isset( $instance['title'] ) ? $instance['title'] : '';
 		$nav_menu = isset( $instance['nav_menu'] ) ? $instance['nav_menu'] : '';
 
 		// Get menus
 		$menus = wp_get_nav_menus();
 
+		$empty_menus_style = $not_empty_menus_style = '';
+		if ( empty( $menus ) ) {
+			$empty_menus_style = ' style="display:none" ';
+		} else {
+			$not_empty_menus_style = ' style="display:none" ';
+		}
```

Comment:
Having this separate from the output is so confusing. Is this related to standards at all?

Reply:
I think this is linked to the backpport of 42217.

Reply:
Updated PR to follow.